src/util/http_utils.js

```diff
@@ -664,6 +665,82 @@ function set_amz_headers(req, res) {
     res.setHeader('x-amz-id-2', req.request_id);
 }
 
+const s3_error_options = {
+    ErrorClass: S3Error,
+    error_missing_content_length: S3Error.MissingContentLength
+};
```

Comment:
Why declare this variable outside of the function?

Reply:
We can remove it. Thanks